modules/500-operator-trivy/crds/aquasecurity.github.io_vulnerabilityreports.yaml

```diff
@@ -0,0 +1,258 @@
+---
```

Comment:
Please add a comment where we can download the new version of the crd.

Reply:
Done.